xml/schema/Event/Event.xml

```diff
@@ -9,10 +9,15 @@
   <field>
     <name>id</name>
     <type>int unsigned</type>
-    <title>Event</title>
+    <title>Event ID</title>
     <required>true</required>
     <comment>Event</comment>
     <add>1.7</add>
+   <pseudoconstant>
+      <table>civicrm_event</table>
+      <keyColumn>id</keyColumn>
+      <labelColumn>title</labelColumn>
+    </pseudoconstant>
```

Comment:
Actually the reason behind this change is to build the Qill for event's id to replace its integer value into event title as mentioned in its psedoconstant property labelColumn = title and function CRM_Contact_BAO_Query::buildQillForFieldValue($daoName, $fieldName, $fieldValue, $op, $pseduoExtraParam = array()) does this job. I was not aware of the regression it would cause to https://github.com/civicrm/civicrm-core/pull/5675 :(